test/build_runtime_jar.sh

```diff
@@ -87,13 +107,15 @@ echo -e "\nbuilt connector name: $SNOWFLAKE_PLUGIN_NAME"
 mkdir -m 777 -p $KAFKA_CONNECT_PLUGIN_PATH || \
 sudo mkdir -m 777 -p $KAFKA_CONNECT_PLUGIN_PATH 
 cp $SNOWFLAKE_PLUGIN_PATH/$SNOWFLAKE_PLUGIN_NAME $KAFKA_CONNECT_PLUGIN_PATH || true
-echo -e "copied connector to $KAFKA_CONNECT_PLUGIN_PATH"
+echo -e "copied SF Plugin Connector to $KAFKA_CONNECT_PLUGIN_PATH"
+
+if [[ $BUILD_FOR_RUNTIME == "confluent" ]]; then
+    echo "For confluent RUNTIME: Copying Kafka Connect Maven Generated Zip file to a temporary location"
+    cp $SNOWFLAKE_PLUGIN_PATH/components/packages/snowflakeinc-snowflake-kafka-connector-*.zip /tmp/sf-kafka-connect-plugin.zip
+    ls /tmp/sf-kafka-connect-plugin*
+fi
 
 KAFKA_CONNECT_DOCKER_JAR_PATH="$SNOWFLAKE_CONNECTOR_PATH/docker-setup/snowflake-kafka-docker/jars"
 mkdir -m 777 -p $KAFKA_CONNECT_DOCKER_JAR_PATH
 cp $SNOWFLAKE_PLUGIN_PATH/$SNOWFLAKE_PLUGIN_NAME $KAFKA_CONNECT_DOCKER_JAR_PATH || true
-echo -e "copied connector to $KAFKA_CONNECT_DOCKER_JAR_PATH for docker"
-
-echo -e "\n===================================== WARNING ====================================="
-echo -e "not recommended to share this jar with customer unless necessary (PrPr feature, etc)"
-echo -e "absolutely do not let the customer run this jar on their production environment\n"
```

Comment:
I thought we added this for a reason?

Reply:
> Thanks Jay. Is the goal here to support running end2end tests against Confluent? Could you update the description to make it more clear?

Ooops, will make it clear.. No, that is not the goal, but this will help us test how our jar/release will behave if we release it on confluent hub.

Reply:
> I thought we added this for a reason?

Not needed anymore, it is not in PrPr..